surefire-its/src/test/resources/surefire-1993-jpms-providing-modules/application/pom.xml

```diff
@@ -59,16 +59,6 @@
     </dependencies>
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <argLine>
-                        --add-modules ALL-DEFAULT
-                        --add-opens application/org.apache.failsafe.its.serviceloader.application=ALL-UNNAMED
-                    </argLine>
-                </configuration>
-            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
The Failsafe still has `--add-opens`?

Reply:
Not sure what you mean... there's no `--add-opens` left here, as far as I can see.

Reply:
ah, ok, I understand that you removed [surefire plugin in the commit](https://github.com/apache/maven-surefire/pull/463/commits/a178700f52d4163deffac51b7bde5d18795c0bf6) so there is now only one plugin - Failsafe without the config. Just fine! I tought that the Surefire was intact - my bad!

Reply:
So, you can now squash the commits and we can make a final review.

Reply:
Just to be sure: squash the commits and then force-push to the same branch?

Reply:
@mthmulders yes

Reply:
Thanks @Tibor17 - squashed and force-pushed.